DCO.md

```diff
@@ -0,0 +1,22 @@
+# JerryScript Developer's Certificate of Origin
+
+The JerryScript project uses the signed-off-by language and process to give us a clear chain of trust for every patch received.
+
+> By making a contribution to this project, I certify that:
+
+> (a)	The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or
+
+> (b)	The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or
+
+> (c)	The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
+
+> (d)	I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project, under the same open source license.
+
+We have the same requirements for using the signed-off-by process as the Linux kernel.
```

Comment:
I don't think we can drop this, DCO.md specifies the terms and conditions of the DCO and the format of the "Signed-off-by" line is part of that. That's what I was referring to when I mentioned that I tried to make DCO.md as small as possible :)  
Ideally we would just include DCO.md into CONTRIBUTING.md but unfortunately GitHub does not support that and there are nos plans to add that in the future (too many security implications).  
I'm afraid we just have to live with the small amount of duplication here.